### ISSUE TITLE
pychromecast  7.1.1-1 breaks mkchromecast

### ISSUE THREAD

**Issue by stefanhh0** [muammar/mkchromecast#322]
After upgrading to pychromecast  7.1.1-1 starting mkchromecast fails as follows:

```
stefan@mars:~$ mkchromecast -n Standardgruppe
Mkchromecast v0.3.8.1
Creating Pulseaudio Sink...
Open Pavucontrol and Select the Mkchromecast Sink.
Starting Local Streaming Server
[Done]
Selected backend: parec
Selected audio codec: mp3
Default bitrate used: 192k
Default sample rate used: 44100Hz.
 * Serving Flask app "mkchromecast.audio" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
PID of main process: 9946
PID of streaming process: 9953
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
Traceback (most recent call last):
  File "/usr/share/mkchromecast/mkchromecast/cast.py", line 113, in _get_chromecasts
    return list(pychromecast.get_chromecasts_as_dict().keys())
AttributeError: module 'pychromecast' has no attribute 'get_chromecasts_as_dict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/mkchromecast", line 283, in <module>
    mk()
  File "/usr/bin/mkchromecast", line 53, in __init__
    self.audio_linux()
  File "/usr/bin/mkchromecast", line 72, in audio_linux
    self.cc.initialize_cast()
  File "/usr/share/mkchromecast/mkchromecast/cast.py", line 135, in initialize_cast
    self.cclist = self._get_chromecasts()
  File "/usr/share/mkchromecast/mkchromecast/cast.py", line 115, in _get_chromecasts
    self._chromecasts_by_name = {c.name: c for c in
  File "/usr/share/mkchromecast/mkchromecast/cast.py", line 115, in <dictcomp>
    self._chromecasts_by_name = {c.name: c for c in
AttributeError: 'list' object has no attribute 'name'
```

After downgrading to:
pychromecast v5.1.0-1 chromecast works agian.

-- System Information:
Debian Release: bullseye/sid
  APT prefers testing
  APT policy: (990, 'testing'), (500, 'testing-debug'), (500, 'unstable')
Architecture: amd64 (x86_64)
Foreign Architectures: i386

Kernel: Linux 5.7.0-1-amd64 (SMP w/4 CPU threads)
Kernel taint flags: TAINT_PROPRIETARY_MODULE, TAINT_OOT_MODULE, TAINT_UNSIGNED_MODULE
Locale: LANG=de_DE.UTF-8, LC_CTYPE=de_DE.UTF-8 (charmap=UTF-8), LANGUAGE=de
Shell: /bin/sh linked to /bin/dash
Init: systemd (via /run/systemd/system)
LSM: AppArmor: enabled

Versions of packages mkchromecast depends on:
ii  flac                  1.3.3-1
ii  gir1.2-notify-0.7     0.7.9-1
ii  lame                  3.100-3
ii  nodejs                10.21.0~dfsg-1
ii  opus-tools            0.1.10-1+b1
ii  python3               3.8.2-3
ii  python3-flask         1.1.2-1
ii  python3-psutil        5.7.0-1
ii  python3-pychromecast  7.1.1-1
ii  python3-pyqt5         5.15.0+dfsg-1+b1
ii  sox                   14.4.2+git20190427-2
ii  vorbis-tools          1.4.0-11
ii  youtube-dl            2020.06.16.1-1

mkchromecast recommends no packages.

Versions of packages mkchromecast suggests:
ii  ffmpeg                   7:4.3-3+b1
pn  libav-tools              <none>
ii  mkchromecast-alsa        0.3.8.1-1
ii  mkchromecast-pulseaudio  0.3.8.1-1

-- no debconf information

**Comment by stefanhh0** [comment]
Also opened two bug reports on debian:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=965249
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=965250

**Comment by stefanhh0** [comment]
Fixed in debian version: mkchromecast/0.3.9~git20200902+db2964a-1
